lib/cf/bridge/src/index.js

```diff
@@ -305,11 +326,14 @@ const reportUsage = (usage, token, config, cb = () => {}) => {
       cb(error, response);
       return;
     }
-    const message = util.format('Failed reporting\n\tusage %j\n\terror %j' +
-      '\n\tresponse %j', usage, error, response);
-    edebug(message);
+    const message = util.format('Failed reporting usage error for the' +
```

Comment:
Shouldn't the message read something like `Failed reporting usage. Consecutive failures: %d`. We don't actually count the failures on each usage but the total consecutive.